app/services/search_payload.rb

```diff
@@ -0,0 +1,36 @@
+class SearchPayload
+  attr_reader :registerable_calendar
```

Comment:
We can probably drop the 'registerable' terminology altogether now that we've moved this away from panopticon, referring to this simply as `calendar`.